tests/fixture/tmpnet/network.go

```diff
@@ -24,222 +23,152 @@ import (
 	"github.com/ava-labs/avalanchego/utils/set"
 )
 
+// The Network type is defined in this file (network.go - orchestration) and
+// network.go (reading/writing configuration).
```

Comment:
is this a typo? Or is there another network.go file I'm missing that this is in reference to?